scripts/module/keybinder.nut

```diff
@@ -15,7 +15,7 @@ function extractFirstWord( input ) {
 
 function onPlayerCommand( player, cmd, text ) {
   cmd = cmd.tolower();
-  if (cmd == "b") {
+  if (cmd == "bind") {
     text = text.tolower();
     local bindTable = {};
     if (text.len() < 3) {
```

Comment:
There should also be an option to list existing bindings. You could repurpose one command:

`/bind <key>` = unbind
`/bind <key> <command[s]>` = bind
`/bind` = help, but also list existing bindings

Reply:
Done and done, thanks!